UI/properties-view.cpp

```diff
@@ -649,6 +649,11 @@ void OBSPropertiesView::AddEditableList(obs_property_t *prop,
 {
 	const char *name = obs_property_name(prop);
 	OBSDataArrayAutoRelease array = obs_data_get_array(settings, name);
+	if (array == NULL) {
+		array = obs_data_array_create();
+		obs_data_set_array(settings, name, array);
+	}
+
```

Comment:
Found the issue. The array was never initialized for the given setting name, so this is the only change I needed to add.

I have verified that new editable lists using scripts or sources (VLC Source and my own plugin) are now working.